packages/react-charting/src/utilities/utilities.ts

```diff
@@ -366,64 +366,69 @@ export const createStringYAxis = (yAxisParams: IYAxisParams, dataPoints: string[
  * @param values
  */
 
-type DataPoint = {
```

Comment:
is DataPoint not used anywhere?

Reply:
It was used only in utilities.

Reply:
Lets keep it in case someone is consuming it externally.